cmd/juju/cloud/updatecredential.go

```diff
@@ -267,19 +268,30 @@ func (c *updateCredentialCommand) updateLocalCredentials(ctx *cmd.Context, updat
 			continue
 		}
 
-		region := cloudCredentials.DefaultRegion
 		if c.Region != "" {
-			region = c.Region
-		}
-		for credentialName, credential := range cloudCredentials.AuthCredentials {
-			verified, err := modelcmd.VerifyCredentials(ctx, aCloud, &credential, credentialName, region)
-			if err != nil {
+			if err := ValidCloudRegion(aCloud, c.Region); err != nil {
 				logger.Errorf("%v", err)
-				ctx.Warningf("Could not verify credential %v for cloud %v locally", credentialName, cloudName)
+				ctx.Warningf("Region %q is not valid for cloud %v.", c.Region, cloudName)
 				erred = true
 				continue
 			}
-			storedCredentials.AuthCredentials[credentialName] = *verified
+		}
```

Comment:
Does this change need a unit test?

Reply:
Added. Although the functionality has been tested in add credential, so additional test a somewhat redundant.